R/plotDots.R

```diff
@@ -111,8 +133,17 @@ plotDots <- function(object, features, group = NULL, block=NULL, exprs_values =
     if (!is.null(max_detected)) {
         evals_long$NumDetected <- pmin(max_detected, evals_long$NumDetected)
     }
-    if (!is.null(max_ave)) {
-        evals_long$Average <- pmin(max_ave, evals_long$Average)
+    if (scale) {
```

Comment:
Did you mean this to be `center`? Arguably the most consistent approach would be to have a `symmetric=center` default in the argument list... assuming that a separate `symmetric` argument even makes sense, see comments above.

I also wonder whether we should have more consistent default color schemes across all these related plots (`plotHeatmap()`, this, and `plotGroupedHeatmap()`). You could imagine creating another utility function that chooses the default color scheme depending on whether `center=TRUE`; if yes, we use whatever palette **pheatmap** uses, which looks nice enough, while for `FALSE`, we can default to viridis.